### PR TITLE
Make viewBox of polygonal board fit intersections

### PR DESCRIPTION
**Pull request by JonKo314**
Before:
grafik

After:
grafik

### CODE REVIEW

File: packages/vue-client/src/components/boards/BadukBoardAbstract.vue [hunk 1]
```diff
@@ -25,6 +26,18 @@ const emit = defineEmits<{
...
 }
+
+const viewBox = computed(() => {
+  const xPositions = props.gamestate.board.Intersections.map(
+    (i) => i.Position.X
+  );
+  const yPositions = props.gamestate.board.Intersections.map(
+    (i) => i.Position.Y
+  );
+  return `${Math.min(...xPositions) - 1} ${Math.min(...yPositions) - 1} ${
```
Comment (merowin) [inline packages/vue-client/src/components/boards/BadukBoardAbstract.vue:37]: At the end of constructing the intersections of the polygonal board, all positions are shifted such that Math.min(...xPositions) and Math.min(...yPositions) are 0, 0. 

However I like that this is handled in the view, so the board is always pictured nicely. So should we remove the abovementioned shifting in the construction algorithm?

**Review by merowin — APPROVED** [review]

File: packages/vue-client/src/components/boards/BadukBoardAbstract.vue (hunk 1 above)
Comment (JonKo314) [reply packages/vue-client/src/components/boards/BadukBoardAbstract.vue:37]: Interesting. It is somewhat redundant, but I don't mind having both, a visualization that can handle negative coordinates and coordinates that are all non-negative.

**Comment by benjaminpjones** [comment]
Anything holding this one back? It looks great, and Im hoping we can avoid merge issues by putting it on `main` sooner than later 😋

**Comment by JonKo314** [comment]
Nope, thanks for the reminder.

**Pull request merged**